### PR TITLE
SUI Atom Badge

### PR DESCRIPTION
**Pull request by jcger**
- [x] Complete migration to theme-basic
- [ ] QA issues
- [x] Raw SVG or wrapped

### CODE REVIEW

File: components/atom/badge/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    height: $p-xl;
+    line-height: $p-xl;
+    padding: 0 $p-m;
+  }
+
+  [class$='icon'] {
+    @include sui-icon--small;
+    display: inline-block;
+    margin-right: $p-s;
+    vertical-align: middle;
```
Comment (beatrizip) [inline components/atom/badge/src/index.scss]: $m-s better

File: components/atom/badge/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  @each $tags-type in $tags {
+
+    $key: nth($tags-type, 1);
+    $value: nth($tags-type, 2);
+
+    &-#{$key} {
+      background-color: $value;
+      color: $c-contrast-AtomBadge ;
```
Comment (beatrizip) [inline components/atom/badge/src/index.scss]: $value is not a descriptive name

File: components/atom/badge/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      fill: $c-contrast-AtomBadge ;
+    }
+
+    &-#{$key}--transparent {
+      background-color: transparent;
+      color: $value;
+      fill: $value;
+    }
```
Comment (beatrizip) [inline components/atom/badge/src/index.scss]: $value is not a descriptive name

File: components/atom/badge/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-AtomBadge: $c-black;
+$c-contrast-AtomBadge: $c-white;
+$bdrs-AtomBadge: $bdrs-m;
```
Comment (JoanClaret) [inline components/atom/badge/src/index.scss]: Should this vars be defined with  `!default` ?

**Review by JoanClaret — APPROVED** [review]

File: components/atom/badge/package.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "latest",
+    "@schibstedspain/sui-theme": "8"
```
Comment (davidbarna) [inline components/atom/badge/package.json]: No need to install sui-theme.

**Review by davidbarna — APPROVED** [review]

**Review by davidbarna — APPROVED** [review]

**Comment by midudev** [comment]
From my point of view, I don't see at all the using boolean propTypes for types and sizes. Any reason to prefer to a `type` and `size` props? I mean, you would need to write the string anyway and you are forced to do a lot of unnecessary checks in the component in order to put the correct one.

**Review by midudev — APPROVED** [review]

**Pull request merged**